Matplotlib.ipynb

```diff
@@ -235,21 +319,21 @@
  ],
  "metadata": {
   "kernelspec": {
-   "display_name": "Python 3",
+   "display_name": "Python 2",
    "language": "python",
-   "name": "python3"
+   "name": "python2"
   },
   "language_info": {
    "codemirror_mode": {
     "name": "ipython",
-    "version": 3
+    "version": 2
    },
    "file_extension": ".py",
    "mimetype": "text/x-python",
    "name": "python",
    "nbconvert_exporter": "python",
-   "pygments_lexer": "ipython3",
-   "version": "3.6.7"
+   "pygments_lexer": "ipython2",
+   "version": "2.7.16"
```

Comment:
@gonuke - do you think it matters that this is stored as python 2 vs 3? We had some kernel issues with running in python 3.

Reply:
It's a little concerning that it's not working in Py3.  Do you know what the issues are?

Reply:
This was I think an issue with @yqin43 's system (couldn't get matplotlib to work or something like that in the notebook I think). It wasn't an issue w/ DAGMC stats.

Reply:
It's probably not the end of the world...

Reply:
we could probably just manually edit the file back to python 3 in these lines..